src/inmanta/moduletool.py

```diff
@@ -1291,10 +1291,17 @@ def _do_update(self, output_directory: str, setup_cfg: ConfigParser, warn_on_mer
         # move plugins or create
         old_plugins = os.path.join(output_directory, "plugins")
         new_plugins = os.path.join(output_directory, "inmanta_plugins", self._module.name)
-        if os.path.exists(old_plugins):
+        if os.path.exists(new_plugins) and os.listdir(new_plugins):
+            raise ModuleBuildFailedError(
+                msg=f"Could not build module: inmanta_plugins/{self._module.name} directory already exists and is not empty"
+            )
+        elif os.path.exists(new_plugins) and os.path.exists(old_plugins):
+            shutil.copytree(old_plugins, new_plugins, dirs_exist_ok=True)
+        elif not os.path.exists(new_plugins) and os.path.exists(old_plugins):
```

Comment:
We've now got four additionall `if` statements. We could get rid of three of them by just deleting the directory (we know to be empty). I think that might be more readable. What do you think?

Reply:
Indeed that might be more readable, I'll change it.